### ISSUE TITLE
migrated to new downloads

### ISSUE THREAD

**Issue by Pyro2677** [Nexus-Mods/Vortex#974]
In the last update i want to say 16.1 i had to change my downloads folder when i opened Vortex and then it created E:\Games\Vortex\Downloads\fallout4 and then 2 folders in here of fallout4 and skyrimse. Not sure why it created the fallout4 and then another 2 folders.

**Comment by Pyro2677** [comment]
Removed the fallout4 directory and pointed Vortex to just the downloads folder, copied all the skyrimse dir and files back. Opened vortex and all my mods are now duplicated with all the copied files in the skyrimse folder now saying not installed with same ones already installed. Tried removing the mods without removing the archive and it doesn't remove them.

**Comment by Anvirol** [comment]
I also had massive issues when initially updating to 0.16 and I was forced to migrate the downloads(archives) folder, but suppose these are only pains for us testers.  

New installations wouldn't have such issues.

I actually had to reinstall whole Vortex and all 200+ mods.

**Comment by TanninOne** [comment]
@Pyro2677 The dialog states quite clearly that "On the next screen, please select an empty directory where all your downloads from Vortex will be placed." It's not asking for each game individual but for one top-level directory for all your downloads. Within that it creates subdirectories for each game.
So you should have selected `E:\Games\Vortex\Downloads`
Now the other part, that too is to be expected. If you go and (re-)move files that are maintained by vortex you're sure to confuse its database. Why you had to manually copy files in the first place isn't clear to me, when you change the download directory inside vortex it automatically moves everything to the new location.

**Comment by Pyro2677** [comment]
Ok firstly I will ignore your condescending reply, and explain why i had to manuall move them. before 16.1 I had the following Dirs E:\Games\Vortex\skyrimse\Downloads, E:\Games\Vortex\skyrimse\Mods, E:\Games\Vortex\skyrimse\Tools. After the update it asked me if i wanted to go to the "new" downloads layout and I clicked yes. It said something along the lines of this could take awhile but it is doing it's thing. Vortex then (by itself) created E:\Games\Vortex\Downloads\Fallout4\skyrimse & E:\Games\Vortex\Downloads\Fallout4\fallout4. It took like 6 goes at doing it because it started to create the folders itself and then would say while merging etc that it couldn't complete because Fallout4 directory wasn't empty. 
I never had made this folder (fallout4)  and Vortex made this on it's own accord in the moving process. Vortex automatically moved the E:\Games\Vortex\skyrimse\Downloads from this location to E:\Games\Vortex\Downloads\Fallout4\skyrimse. So i didn't get to choose anything.

After deleting the folders and restarting Vortex it tried again & Again same issue. After a few more tries i rebooted and tried again and it still didn't do it. I had to MANUALLY copy the files into this newly created Downloads folder BECAUSE VORTEX wouldn't do it. 

At this point i was still confused because i never had Vortex handle my Fallout4 install. This was done using MO2. So why would it try to create Fallout4 folders?

> "Why you had to manually copy files in the first place isn't clear to me, when you change the download directory inside vortex it automatically moves everything to the new location."

 So this is the reason i had to manually move folders/files because Vortex wouldn't!!! 

Now lets say for arguments sake that I'm not a complete moron and i chose the Downloads directory. WHY did it create a Fallout4 folder and inside that it created 2 more subfolders when i didn't even have Vortex handling fallout 4 in the first place? Now you say that you just point vortex to a main and it creates them all for you well I have decided to use Vortex to handle my Fallout 4 and when i created the profile and i started to install mods it started to put all the mods into E:\Games\Vortex\fallout4 NOT E:\Games\Vortex\fallout4\Mods like it did when i created a Skyrimse profile. Now i tried to move all mods to E:\Games\Vortex\fallout4\Mods. SO i made subfolders same as Skyrimse and yes empty mods folder. Now when i pointed to the Mods folder and hit Apply it created E:\Games\Vortex\fallout4\Mods\mods and said it can't move the files because the directory isn't empty. then I tried again and just chose the fallout4 folder but it copied them back into this folder. I created Mods folder again and clicked apply and it was saying this folder is in use by another program. I eventually after about 40mins trying to get VBortex to move the damn directory i just pointed it to another location on my hard drive and created the fallout4/Mods & Tools subfolders and them got Vortex to move them back and this time it worked.
